basxconnect/core/urls.py

```diff
@@ -146,4 +146,5 @@
     ),
     autopath(person_modals_views.EditPostalAddressView.as_view(model=models.Postal)),
     autopath(person_modals_views.AddPostalAddressView.as_view(model=models.Postal)),
+    autopath(person_modals_views.DeletePostalAddressView.as_view(model=models.Postal)),
```

Comment:
In case you were wondering: autopath supports a parameter ```urlname``` which can be a string to give a custom name to a URL which can then be resolved with ```resolve(urlname)```. This *can* make sense if you are creating just a single-use AJAX view or similar. In the default case it is just automatically generated, but it can be cumbersome to find the correct urlname via ```./manage.py show_urls``` or similar.

Reply:
how to you like to idea of storing the "name" of the view inside the view class itself? I think it would be nice to have this string defined only ad one place and referenced everywhere else so that we can avoid typos

Reply:
As I mentioned in the other comment: I don't really see a good reason for this. Typos happen, but they happen everywhere. In this case they get caught as soon as you try to resolve the URL. It is all the same possible that we make a typo when writing the variable name and due to the dynamic nature of python it might still not get caught by the IDE or when starting the server (e.g. when the call to reverse is inside a lambda).
I think the name of the view-object (be it a function or a class) should be enough in many cases. And there autopath does its job pretty well (although, it could be improved I guess, but URL names also need to be unique, so it is not that trivial). In cases where we want to explicitly (or manually) name a URL we should do so in the urls.py.
Also keep in mind that a view can be used on different URLs which can be useful at times. If a view is used on more than one URL we have to name it manually anyway.

All in all I am against introducing a third way to name URLs, in addition to the standard "manual" way and the autopath generator.